### PR TITLE
Release 5.0.0

### PR DESCRIPTION
**Pull request by aweibel**
This is a major release because of the restructuring we did with the Sketch files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,41 +2,50 @@
...
 </p>
-<p align="center">Install and always stay up to date by subscribing to the Sketch Library links below.
-<br />
-<strong>» Copy/paste the URLs below in any browser address bar «</strong></p>
-<ol>
-<li><b>Component Library</b>
+<strong>» To install, copy/paste the URLs below in any browser address bar «</strong></p>
+<i>Required libraries</i>
+<ul>
+<li><b>SLDS Icons</b>
```
Comment (liselatorre) [inline README.md]: We should be consistent with the use of strong vs b if possible. Also, if we are bolding or italicizing to act as a header, h tags might be better from an accessibility standpoint

File: README.md [hunk 2]
```diff
@@ -2,41 +2,50 @@
...
 </p>
 <br />
-<h1 align="center" style="border-bottom:none;">Salesforce Lightning Design System (SLDS) UI Kit</h1>
-<p align="center">
-The SLDS UI Kit is a collection of Sketch Libraries of <a href="https://www.lightningdesignsystem.com">Lightning Design System</a> components and design patterns. The thousands of customers and partners who build applications on the Salesforce platform can use these resources to quickly visualize and test designs before they're implemented in their Salesforce organizations.
+<h1 style="border-bottom:none;">Salesforce Lightning Design System (SLDS) UI Kit</h1>
+<p>
+The Salesforce Lightning Design System (SLDS) UI Kit is a collection of resources to support designing and prototyping using <a href="https://www.lightningdesignsystem.com">Lightning Design System</a> components, tokens, and design patterns. There are also useful resources to help make design workflows more efficient with artifacts like page templates, wireframes, key product screens, and components for writing specifications.
```
Comment (liselatorre) [inline README.md]: Do we want a target=_blank on the link?

File: README.md [hunk 3]
```diff
@@ -2,41 +2,50 @@
...
-----
 
 ## Requirements
 
 ### Sketch
 
 Download the most recent version of [Sketch](https://www.sketchapp.com/).
 <br />
-<i>These files are optimized for Sketch but if you don't have it you can use a free application with less functionality called <a href="https://www.figma.com/" target="_blank" title="Figma">Figma</a> instead</i>
+<i>If you do not have Sketch, you can use the <a href="https://github.com/salesforce-ux/design-system-ui-kit-framerx" target="_blank" title="Framer SLDS UI Kit">Framer SLDS UI Kit</a> or <a href="https://www.figma.com/file/5dgFdCHB6FGjfOPAZEDNVK/Lightning-Design-System-Components-for-Web?node-id=0%3A1" target="_blank" title="Figma SLDS Components for Web">Figma SLDS Components for Web</a> and <a href="https://www.figma.com/file/5dgFdCHB6FGjfOPAZEDNVK/Lightning-Design-System-Components-for-Web?node-id=0%3A1" target="_blank" title="Figma SLDS Icons">Figma SLDS Icons</a> libraries, however, they may not be as up-to-date as the Sketch files.</i>
```
Comment (liselatorre) [inline README.md]: `em` gives us the same styles as italics while conveying meaning to folks on screen readers

File: README.md [hunk 4]
```diff
@@ -48,121 +57,119 @@ Download the most recent version of [Sketch](https://www.sketchapp.com/).
...
-   * <code>sketch://add-library?url=https://www.lightningdesignsystem.com/sketch-library-rfl-rss.xml</code>
-5. User Engagement Patterns (optional)
-   * <code>sketch://add-library?url=https://www.lightningdesignsystem.com/sketch-library-user-engagement-rss.xml</code>
+    sketch://add-library?url=https://www.lightningdesignsystem.com/sketch-library-rss.xml
+**SLDS Components for Web**
+    
+    sketch://add-library?url=https://www.lightningdesignsystem.com/sketch-library-icons-rss.xml
 
-Alternatively, download the Sketch Libraries in this repository you'd like to install. Open `Sketch > Preferences > Libraries` and click `Add Library...`. Select the library you'd like to install.
+Alternatively, download the Sketch Libraries in this repository you'd like to install. Open `Sketch > Preferences > Libraries` and click `Add Library...`. Select the library you'd like to install. If you do this method, you'll have to frequently come back to this repository, download new version, and repeat this installation. This method is not recommended because it's such a hassle.
```
Comment (liselatorre) [inline README.md]: ...`download the new version`...

Comment (aweibel) [reply README.md]: Updated!

File: README.md (hunk 3 above)
Comment (aweibel) [reply README.md]: Updated!

File: README.md (hunk 2 above)
Comment (aweibel) [reply README.md]: Updated!

File: README.md (hunk 1 above)
Comment (aweibel) [reply README.md]: Updated!

**Pull request merged**